django/contrib/postgres/lookups.py

```diff
@@ -46,12 +46,36 @@ class HasAnyKeys(HasKeys):
     operator = '?|'
 
 
+class LowerInclusive(Transform):
+    lookup_name = 'lower_inc'
+    function = 'LOWER_INC'
+    output_field = BooleanField()
+
+
+class LowerInfinite(Transform):
+    lookup_name = 'lower_inf'
+    function = 'LOWER_INF'
+    output_field = BooleanField()
+
+
 class Unaccent(Transform):
     bilateral = True
     lookup_name = 'unaccent'
     function = 'UNACCENT'
 
 
+class UpperInclusive(Transform):
```

Comment:
Please use `@RangeField.register_lookup` and move these transforms to `django/contrib/postgres/fields/ranges.py`.

Reply:
got it